src/twisted/python/_release.py

```diff
@@ -39,14 +40,66 @@
 
 
 def runCommand(args, **kwargs):
-    """Execute a vector of arguments.
+    """
+    Execute a vector of arguments.
 
     This is a wrapper around L{subprocess.check_output}, so it takes
     the same arguments as L{subprocess.Popen} with one difference: all
     arguments after the vector must be keyword arguments.
+
+    @param args: arguments passed to L{subprocess.check_output}
+    @param kwargs: keyword arguments passed to L{subprocess.check_output}
+    @return: command output
+    @rtype: L{str}
     """
     kwargs['stderr'] = STDOUT
-    return check_output(args, **kwargs)
+    output = check_output(args, **kwargs)
+
+    # The check_output() function always returns bytes
+    # on Python 2 and 3, but we want this function to always return
+    # str.
+    if not isinstance(output, str):
+        output = output.decode("utf-8")
+    return output
+
+
+
+class FilePathStrContent(FilePath):
+    """
+    Wrapper around L{FilePath} to make dealing with
+    L{bytes} and L{str}
```

Comment:
I think it's worth keeping in mind that this is
1. private API, not public
2. used only as a _tool_, not even indirectly exposed to applications.

I agree that this is not the best approach, in that it adds additional wrapper code bloat rather than dealing with the type issues at each call site, but it seems like it would be expedient to land this and address the review feedback as small follow-ups, as there's no compatibility burden associated with the sub-optimal solution here.